packages/integration-tests/src/test-projects/data-storage-auth-with-triggers-ts/amplify/func-src/handler_with_aws_sqs.ts

```diff
@@ -0,0 +1,32 @@
+import { SQSClient, SendMessageCommand } from '@aws-sdk/client-sqs';
+
+/**
+ * This function asserts that schedule functions are working properly
+ */
+export const handler = async () => {
+  const sqsClient = new SQSClient({ region: process.env.region });
+
+  const queueUrl = process.env.SQS_QUEUE_URL;
+
+  if (!queueUrl) {
+    throw new Error('SQS_QUEUE_URL is not set in environment variables');
+  }
+
+  try {
+    const messageBody = JSON.stringify({
+      message: 'Test message from Lambda',
+      timeStamp: new Date().toISOString(),
+    });
+
+    await sqsClient.send(
+      new SendMessageCommand({
+        QueueUrl: queueUrl,
+        MessageBody: messageBody,
+      })
+    );
+
+    return 'It is working';
+  } catch (err) {
+    throw err;
+  }
```

Comment:
this try catch doesn't do much and can be removed.

Reply:
Done in https://github.com/aws-amplify/amplify-backend/pull/1778/commits/e414ee765fdfd3f33553252babcecd69f0299f1e